### PR TITLE
Feat/add openlogin v5 and sapphire updates

### PR DESCRIPTION
**Pull request by grvgoel81**
Features:
* Adding Sapphire networks
* Adding openlogin v5 support.

### CODE REVIEW

File: core/src/main/java/com/web3auth/core/types/MfaSetting.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.web3auth.core.types
+
+data class MfaSetting(
+    var enable: Boolean?,
```
Comment (chaitanyapotti) [inline core/src/main/java/com/web3auth/core/types/MfaSetting.kt:4]: enable is mandatory, priority is optional, mandatory is optional

File: core/src/main/java/com/web3auth/core/Web3Auth.kt [hunk 2]
```diff
@@ -171,7 +213,7 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
      */
     private fun authorizeSession(): CompletableFuture<Web3AuthResponse> {
         val sessionCompletableFuture: CompletableFuture<Web3AuthResponse> = CompletableFuture()
-        val sessionResponse: CompletableFuture<String> = sessionManager.authorizeSession(true)
+        val sessionResponse: CompletableFuture<String> = sessionManager.authorizeSession(false)
```
Comment (chaitanyapotti) [inline core/src/main/java/com/web3auth/core/Web3Auth.kt:216]: why is this false now?

File: core/src/main/java/com/web3auth/core/Web3Auth.kt [hunk 3]
```diff
@@ -32,44 +32,65 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
...
-            customTabs.intent.setPackage(defaultBrowser)
-            customTabs.launchUrl(context, url)
-        } else if (customTabsBrowsers.isNotEmpty()) {
-            val customTabs = CustomTabsIntent.Builder().build()
-            customTabs.intent.setPackage(customTabsBrowsers[0])
-            customTabs.launchUrl(context, url)
-        } else {
-            // Open in browser externally
-            context.startActivity(Intent(Intent.ACTION_VIEW, url))
+                println("url: $url")
```
Comment (chaitanyapotti) [inline core/src/main/java/com/web3auth/core/Web3Auth.kt:74]: remove print

File: core/src/main/java/com/web3auth/core/types/MfaSetting.kt (hunk 1 above)
Comment (grvgoel81) [reply core/src/main/java/com/web3auth/core/types/MfaSetting.kt]: Resolved

File: core/src/main/java/com/web3auth/core/Web3Auth.kt (hunk 3 above)
Comment (grvgoel81) [reply core/src/main/java/com/web3auth/core/Web3Auth.kt]: Resolved

File: core/src/main/java/com/web3auth/core/Web3Auth.kt (hunk 2 above)
Comment (grvgoel81) [reply core/src/main/java/com/web3auth/core/Web3Auth.kt:214]: Since encryption and ecryption happening in native android side.

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**